RSCore/Sources/RSCore/Shared/String+RSCore.swift

```diff
@@ -195,7 +195,15 @@ public extension String {
 	///
 	/// - Note: Doesn't work correctly with nested tags of the same name.
 	private func removingTagAndContents(_ tag: String) -> String {
-		return self.replacingOccurrences(of: "<\(tag).+?</\(tag)>", with: "", options: [.regularExpression, .caseInsensitive])
+		let pattern = "<\(tag)>.*?<\\/\(tag)>"
+		if let regex = try? NSRegularExpression(pattern: pattern, options: [.dotMatchesLineSeparators, .caseInsensitive]) {
+			let range = NSRange(location: 0, length: self.utf16.count)
+			let modifiedString = regex.stringByReplacingMatches(in: self, options: [], range: range, withTemplate: "")
+			return modifiedString
+		} else {
```

Comment:
Would you add a comment here explaining the approach? I’m no regex expert, and it would be helpful to me. (One question I have: wouldn’t there be a way to alter the pattern we were already using to include line separators?)